guides/source/association_basics.md

```diff
@@ -761,17 +761,60 @@ class Book < ApplicationRecord
 end
 ```
 
-Active Record will attempt to automatically identify that these two models share a bi-directional association based on the association name. In this way, Active Record will only load one copy of the `Author` object, making your application more efficient and preventing inconsistent data:
-
-```irb
-irb> a = Author.first
-irb> b = a.books.first
-irb> a.first_name == b.author.first_name
-=> true
-irb> a.first_name = 'David'
-irb> a.first_name == b.author.first_name
-=> true
-```
+Active Record will attempt to automatically identify that these two models share
+a bi-directional association based on the association name. This information
+allows Active Record to:
+
+* Prevent needless queries for already-loaded data
+
+  ```irb
+  irb> author = Author.first
+  irb> author.books.all? do |book|
+  irb>   book.author.equal?(author) # No additional queries executed here
+  irb> end
+  => true
+  ```
```

Comment:
Unfortunately, markdown doesn't recognize the code block as being part of the list item if it only has a 2-space indent.  It has to be at least 4 spaces:

```suggestion
    ```irb
    irb> author = Author.first
    irb> author.books.all? do |book|
    irb>   book.author.equal?(author) # No additional queries executed here
    irb> end
    => true
    ```
  ```
```

Reply:
🤦🏻 Thanks for spotting that!